src/Stratis.Bitcoin/P2P/Protocol/Payloads/VersionPayload.cs

```diff
@@ -172,21 +172,21 @@ public bool Relay
             }
         }
 
-        private VarString user_agent;
+        private VarString userAgentVarString;
```

Comment:
Not sure I like the `VarString` suffix as this style of variable naming (including the variable type) is outdated.

Reply:
Yeah, the issue is because there is another `userAgent` field. Let me see if I can fix this better.

Reply:
OK fixed 👍